src/components/App.js

```diff
@@ -19,7 +19,13 @@ function App(props) {
           </p>
           <p>
             For more information and options please{" "}
-            <a href="https://discord.gg/Bpzfsgx">visit our Discord community</a>
+            <a
+              href="https://discord.gg/Bpzfsgx"
```

Comment:
We need to switch this to the new Discord (chat.tbtc.network)

Reply:
Donezo.